subworkflows/local/amp.nf

```diff
@@ -0,0 +1,26 @@
+/*
+    Run AMP screening tools
+*/
+
+include { MACREL_CONTIGS          } from '../../modules/nf-core/modules/macrel/contigs/main'
+
+workflow AMP {
+    take:
+    contigs // file: /path/to/samplesheet.csv
+
+    main:
+    ch_versions = Channel.empty()
+    ch_mqc      = Channel.empty()
+
+    // TODO AMPEP(?)
+    // TODO ampir
+    if ( !params.skip_amp_macrel ) {
+        MACREL_CONTIGS ( contigs )
+        ch_versions = ch_versions.mix(MACREL_CONTIGS.out.versions)
+    }
+
+    emit:
+    versions = ch_versions
+    mqc = ch_mqc
```

Comment:
Should the output of MACREL_CONTIGS not be mixed into `ch_mqc`?

Reply:
There is currently no MQC module for MACREL so better to not pollute MQC work directory